File: Java/Practice/Program4.java
import java.util.Scanner;

public class Program4
{
    public static void main(String args[])
    {
        float fSum;
        float iValue1;
        float iValue2;

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter First Number : ");
         iValue1= sc.nextFloat();

        System.out.println("Enter Second Number : ");
         iValue2 = sc.nextFloat();

        fSum = iValue1 + iValue2;
        System.out.println("Addition is : "+fSum);

        sc.close();
    }
}

File: C/Practice/program4.c
#include<stdio.h>

int main()
{
    float fValue1 = 0.0f;                  //  To Store First input
    float fValue2 = 0.0f;                  //  To Store Second input
    float fResult = 0.0f;                  //  To Store the Result

    printf("Enter First Number : \n");
    scanf("%f",&fValue1);

    printf("Enter Second Number : \n");
    scanf("%f",&fValue2);

    fResult = fValue1 + fValue2;            //  Perform the Addition
    printf("Addition is : %f\n",fResult);
    return 0;
}